src/UI/Spinner/Spinner.jsx

```diff
@@ -0,0 +1,15 @@
+import './Spinner.css';
+
+const Spinner = () => {
+  return (
+    <div className={['loader', 'loader--fan-out'].join(' ')}>
+      <div className={['item', 'item-1'].join(' ')} />
+      <div className={['item', 'item-2'].join(' ')} />
+      <div className={['item', 'item-3'].join(' ')} />
+      <div className={['item', 'item-4'].join(' ')} />
+      <div className={['item', 'item-5'].join(' ')} />
```

Comment:
Why complexify class names declarations ? 
`{['loader', 'loader--fan-out'].join(' ')}` can be replaced by `"loader loader--fan-out"`

Reply:
problem fixed